source/conf.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python
+
+
+# Fedora Malaysia build configuration file, created by
+# `ablog start` on Thu Dec 17 14:06:09 2020.
+#
+# Note that not all possible configuration values are present in this file.
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import os
+import sys
+import ablog
+import alabaster
+
+# -- General ABlog Options ----------------------------------------------------
+
+# A path relative to the configuration directory for blog archive pages.
+# blog_path = 'blog'
+
+# The "title" for the blog, used in active pages.  Default is ``'Blog'``.
+blog_title = u'Fedora Malaysia Blog'
+
+# Base URL for the website, required for generating feeds.
+# e.g. blog_baseurl = "http://example.com/"
+blog_baseurl = u''
+
+# Choose to archive only post titles. Archiving only titles can speed
+# up project building.
+# blog_archive_titles = False
+
+# -- Blog Authors, Languages, and Locations -----------------------------------
+
+# A dictionary of author names mapping to author full display names and
+# links. Dictionary keys are what should be used in ``post`` directive
+# to refer to the author.  Default is ``{}``.
+blog_authors = {
+    'Robbi Nespu': ('Robbi Nespu', None),
+}
```

Comment:
Need changes.

Reply:
I not sure what changes should i do here

Reply:
Sorry, not this one. Refer to comment on index.rst.